.circleci/config.yml

```diff
@@ -39,16 +39,17 @@ jobs:
 
   test:
     docker:
-    - image: cimg/python:3.10
+    - image: cimg/python:3.12
     working_directory: ~/repo
     steps:
     - checkout
     - restore_cache:
         keys:
         - v6-dependencies-{{ checksum "Pipfile.lock" }}
     - run: *install-pipenv
+    - run: pipenv run pipenv verify
     - run: pipenv run ruff-check
-    - run: pipenv run ruff-format
+    - run: pipenv run djhtml --check .
```

Comment:
I initially wanted to add

```
- run: pipenv run python manage.py check
- run: pipenv run python manage.py makemigrations --check
```

here, but because we don't run with any database configured in CI these fail. I am going to stop pulling the thread here for this PR but it might be useful to add that in future.